docs/source/tutorials/qaoa.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    c = QAOAansatz(params=params[0], g=example_graph, circuit=True)
</pre><p>write a for loop for results of different circuit, or only visulize the result for the best circuit</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>